pkgs/applications/networking/browsers/otter/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, qt5, cmake, openssl, gst_all_1, fetchFromGitHub
+, version ? "0.9.94"
+, sourceSha ? "19mfm0f6qqkd78aa6q4nq1y9gnlasqiyk68zgqjp1i03g70h08k5"
+}:
+stdenv.mkDerivation {
+  name = "otter-browser-${version}";
+
+  src = fetchFromGitHub {
+    owner = "OtterBrowser";
+    repo = "otter-browser";
+    rev = "v${version}";
+    sha256 = sourceSha;
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = with qt5; [ qtbase qtmultimedia qtwebengine ];
+
+  meta = with stdenv.lib; {
+    license = licenses.gpl3;
```

Comment:
Should be `gpl3Plus`. See for example https://github.com/OtterBrowser/otter-browser/blob/master/src/main.cpp#L9.